sos/report/plugins/iprconfig.py

```diff
@@ -32,6 +32,7 @@ def setup(self):
             "iprconfig -c show-af-disks",
             "iprconfig -c show-all-af-disks",
             "iprconfig -c show-slots",
+            "iprconfig -c dump"
```

Comment:
Can we get a confirmation on @bmr-cymru 's earlier concern about this:

> Ack so long as it's just the iprconfig -c dump command (and not the iprsos wrapper script that appears to duplicate a bunch of information from sos).

If this is just a separate command then ack, but if it's a sos-like script we'd want to drop this and hopefully add the actual collections it is doing to sos natively.

Reply:
Do we need to collect output of following commands instead of adding iprconfig -c dump?
iprconfig -c show-alt-config
iprconfig -c show-config
iprconfig -c show-ioas
iprconfig -c show-arrays
iprconfig -c show-hot-spares
iprconfig -c show-jbod-disks
iprconfig -c show-details
iprconfig -c show-battery-info
iprconfig -c query-path-status
iprconfig -c query-path-details
iprconfig -c show-perf

following commands which are already added to sos/report/plugins/iprconfig.py
 "iprconfig -c show-af-disks",
"iprconfig -c show-all-af-disks",
"iprconfig -c show-slots",

"iprconfig -c dump"  will collect  output of all the above mentioned commands

Reply:
Using `dump` if it's just a shorthand for those commands is fine - the only concern was whether it might also run the `iprsos` script in the background (which is more of a problem since it duplicates `sos` functionality).

Reply:
Here is definition of "dump" argument of iprconfig command
dump
Display detailed hardware and system information on standard output. In case a report file is needed, the iprsos command will create one at /var/log/iprsos.log.
Example:
iprconfig -c dump

Reply:
That was the comment in the docs that confused me - it wasn't clear whether that implied it would be created automatically or if the user is supposed to run `iprsos` only if one is needed...

Reply:
ok, got the system today and verified once again iprconfig -c dump will not run the iprsos script in the background and it will not create /vat/log/iprsos.log file, it will just dump output of all the commands which are mentioned above